src/libcore/sync/atomic.rs

```diff
@@ -1119,8 +1131,15 @@ macro_rules! atomic_int {
 
         #[$stable_from]
         impl From<$int_type> for $atomic_type {
-            #[inline]
-            fn from(v: $int_type) -> Self { Self::new(v) }
+            doc_comment! {
```

Comment:
```
            doc_comment! {
                concat!(
"Converts an [`", stringify!($int_type), "`] into an [`", stringify!($atomic_type), "`].

[`", stringify!($int_type), "`]: ../../../std/primitive.", stringify!($int_type), ".html
[`", stringify!($atomic_type), "`]: struct.", stringify!($atomic_type), ".html"),
                #[inline]
                fn from(v: $int_type) -> Self { Self::new(v) }
            }
        }
```

Reply:
Hey mate I don't see the differences, I did a copy and paste and it doesn't show any diff.

Reply:
It is very strange that it used to be green in my old commits, which I reset 😞